### ISSUE TITLE
Block validation is poor in the latest wallet

### ISSUE THREAD

**Issue by NotoriousPyro** [mvs-org/metaverse#252]
I used to use the old version of the wallet which most pools use yet it had high network and CPU usage associated with it.

In the most recent version, I have been using it and during a network DDoS our blocks are all submitted and returned "true" for being valid...

Now my pool manages to rack up a crap load of orphans during these which ruins our orphan rate, while other pools find nothing yet it appears "positive" to them since their orphan rate is unaffected.

**Comment by betachen** [comment]
I see.
Can you send us the version of recent wallet ?

after v0.7.3, we can add IPv4 address in the blacklist in `mvs.conf` file, that would be useful the defense the DDoS manually.

**Comment by NotoriousPyro** [comment]
Can you tell me where to send it?

**Comment by NotoriousPyro** [comment]
If you just want "versions" then:
```
{
        "settings" :
        {
                "network.channel_handshake_seconds" : "30",
                "network.connect_retries" : "0",
                "network.connect_timeout_seconds" : "5",
                "network.debug_file" : "debug.log",
                "network.error_file" : "error.log",
                "network.hosts_file" : "hosts.cache",
                "network.identifier" : "3652501241",
                "network.seeds" : "main-asia.metaverse.live:5251,main-americas.metaverse.live:5251,main-europe.metaverse.live:5251,main-asia.mvs.live:5251,main-americas.mvs.live:5251,main-europe.mvs.live:5251,seed.getmvs.org:5251",
                "server.client_private_key" : "0000000000000000000000000000000000000000",
                "server.connect_retries" : "0",
                "server.connect_timeout_seconds" : "5",
                "server.server_public_key" : "0000000000000000000000000000000000000000",
                "server.url" : "tcp://127.0.0.1:9091",
                "wallet.hd_public_version" : "76067358",
                "wallet.hd_secret_version" : "76066276",
                "wallet.pay_to_public_key_hash_version" : "0",
                "wallet.pay_to_script_hash_version" : "5",
                "wallet.transaction_version" : "1",
                "wallet.wif_version" : "128"
        }
}
```

```
{
        "database-version" : "0.6.2",
        "difficulty" : "6990588802452",
        "hash-rate" : 0,
        "height" : 982602,
        "is-mining" : false,
        "network-assets-count" : 181,
        "peers" : 10,
        "protocol-version" : 70012,
        "testnet" : false,
        "wallet-account-count" : 2,
        "wallet-version" : "0.7.3"
}
```

**Comment by betachen** [comment]
Thanks, I mean just the version information.
can you find out the attacker IP address, then add them in `mvs.conf`
```
[network]
blacklist=$attacker_ip_address
```

**Comment by NotoriousPyro** [comment]
This is network-wide transactional DDoS. How would one do this?

**Comment by NotoriousPyro** [comment]
Also, I think the block validation being poor is due to the removal of `fetchheaderext` which was resolved in #253

**Comment by NotoriousPyro** [comment]
Will reopen if needed.
